dags/quicksuggest2bq.py

```diff
@@ -0,0 +1,45 @@
+"""
```

Comment:
How about we name this DAG a bit more generically? Perhaps `contextual_services_import`? There could be other import tasks we would want to run on this schedule.